### ISSUE TITLE
InvalidArgumentError (see above for traceback): Not enough time for target transition sequence

### ISSUE THREAD

**Issue by zhouyikang** [nl8590687/ASRT_SpeechRecognition#33]
Caused by op u'ctc/CTCLoss', defined at:
  File "train_mspeech.py", line 44, in <module>
    ms = ModelSpeech(datapath)
  File "/home/yuyin/ASRT_SpeechRecognition-master/SpeechModel24.py", line 39, in __init__
    self._model, self.base_model = self.CreateModel()
  File "/home/yuyin/ASRT_SpeechRecognition-master/SpeechModel24.py", line 109, in CreateModel
    loss_out = Lambda(self.ctc_lambda_func, output_shape=(1,), name='ctc')([y_pred, labels, input_length, label_length])
  File "/usr/local/lib/python2.7/dist-packages/keras/engine/base_layer.py", line 457, in __call__
    output = self.call(inputs, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/keras/layers/core.py", line 682, in call
    return self.function(inputs, **arguments)
  File "/home/yuyin/ASRT_SpeechRecognition-master/SpeechModel24.py", line 136, in ctc_lambda_func
    return K.ctc_batch_cost(labels, y_pred, input_length, label_length)
  File "/usr/local/lib/python2.7/dist-packages/keras/backend/tensorflow_backend.py", line 4167, in ctc_batch_cost
    sequence_length=input_length), 1)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/ctc_ops.py", line 158, in ctc_loss
    ignore_longer_outputs_than_inputs=ignore_longer_outputs_than_inputs)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gen_ctc_ops.py", line 283, in ctc_loss
    name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 3290, in create_op
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1654, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InvalidArgumentError (see above for traceback): Not enough time for target transition sequence (required: 15, available: 13)10You can turn this error into a warning by using the flag ignore_longer_outputs_than_inputs
         [[Node: ctc/CTCLoss = CTCLoss[ctc_merge_repeated=true, ignore_longer_outputs_than_inputs=false, preprocess_collapse_repeated=false, _device="/job:localhost/replica:0/task:0/device:CPU:0"](ctc/Log/_285, ctc/ToInt64/_287, ctc/ToInt32_2/_289, ctc/ToInt32_1/_291)]]
         [[Node: ctc/CTCLoss/_293 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device_incarnation=1, tensor_name="edge_889_ctc/CTCLoss", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:GPU:0"]()]]

**Comment by nl8590687** [comment]
请问您是不是使用了新数据？还有就是，我看到您是使用了python2.7来运行的，请使用python3来运行，本项目代码不支持python2，在2.7下运行会出现各种奇奇怪怪的问题
